app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,30 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @post = Post.where(author_id: @user.id).order(created_at: :desc)
+    @post = Post.where(user_id: @user.id).order(created_at: :desc)
+  end
```

Comment:
- [ ] Your web app runs with errors. The column user_id does not exist in the posts table. Please fix that.

<img width="516" alt="image" src="https://user-images.githubusercontent.com/77190203/179350648-0b488197-d17d-4aef-b724-f44ba86c7113.png">